recipe/gen_patch_json.py

```diff
@@ -1454,6 +1454,13 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
                                     if not c.startswith("qt ")]
             record["constrains"].append("qt 5.15.2|5.15.3|5.15.4")
 
+        # tifffile 2022.2.2 and more recent versions requires python >=3.8.
+        # See https://github.com/conda-forge/tifffile-feedstock/issues/93
+        # Fixed in https://github.com/conda-forge/tifffile-feedstock/pull/94
+        if record_name == "tifffile":
+            if record["version"] >= "2022.2.2" and record["version"] < "2022.4.26":
```

Comment:
```suggestion
            if record["version"] >= "2022.2.2" and (record["version"] < "2022.4.26" and record["build_number"] < ....):
```

can we be a little more specific with the build number too.

Reply:
This should be all builds and for these versions (2022.2.2 to 2022.4.26, excluded), there is only a single build per release: https://anaconda.org/conda-forge/tifffile/files. In this case, is it worth being specific about the build number?

Reply:
you are correct thank you.